libs/ui-toolkit/src/components/drawer/drawer.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { theme } from '@vegaprotocol/tailwindcss-config';
+import type { ReactElement } from 'react';
+import { useEffect } from 'react';
+import Drawer from '@mui/material/Drawer';
```

Comment:
fix (blocking): I'd prefer to not add @mui components to the toolkit.

I recently created a drawer for the token dapp using radix. I didn't do it in the ui-toolkit as I didn't think it was needed across projects, until now! 😄 Its not merged yet but you can [find it here](https://github.com/vegaprotocol/frontend-monorepo/pull/423/files#diff-9e246d25dc7e309c6afe507168fa7155fecfe9d316f8356fc500b04e772afdefR177)

It doesn't cover as much functionality as your implementation does but perhaps its a useful starting point for migrating this from using @mui to @radix-ui.

Alternatively, if you want to just roll with @mui for now, can you make this a local component for the simple-trading-app.

Reply:
Moved into simple trading project for now after discussion. Future ticket will be created to refactor into tailwind ui and then add back.